### PR TITLE
Minor enhancment ARM devtools add docker tool

### PR DESCRIPTION
**Pull request by microtechno9000**
# Description
ARM Developer tools, added command that allows for docker management

`./armdevtools.py -dr /home/arm/armdocker.bash `
```
./armdevtools.py -h
usage: armdevtools.py [-h] [-b B] [-d] [-dr DR] [-db_rem] [-db_data] [-qa] [-pr] [-v]

Automatic Ripping Machine Development Tool Scripts

options:
  -h, --help  show this help message and exit
...
  -dr DR      Docker rebuild post ARM code update. Requires docker run script path to run.
...
```

Fixes # n/a

## Type of change
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# How Has This Been Tested?
Ubuntu 20.04

- [x] Docker
- [x] Ubuntu 20.04

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have tested that my fix is effective or that my feature works

# Changelog:
as above, added docker command to armdevtools

# Logs
Attach logs from successful test runs here
[armdevtool_log.txt](https://github.com/automatic-ripping-machine/automatic-ripping-machine/files/10956849/armdevtool_log.txt)

### CODE REVIEW

**Comment by shitwolfymakes** [comment]
Please add documentation (examples) for each use case, forgot to request this last time.

**Comment by microtechno9000** [comment]
There is a readme in the armdevtools folder, do you want this added to the Wiki?

**Comment by shitwolfymakes** [comment]
The README is a good overview of the capability, but is insufficient. For full tutorials the wiki would be a better place.

**Comment by microtechno9000** [comment]
Details added to the wiki
https://github.com/automatic-ripping-machine/automatic-ripping-machine/wiki/ARM-Development-Tools

**Comment by shitwolfymakes** [comment]
Great work on the documentation!

File: devtools/armdevtools.py [hunk 1]
```diff
@@ -3,76 +3,40 @@
...
-
 # Commence reading from the input options
 desc = "Automatic Ripping Machine Development Tool Scripts"
 parser = argparse.ArgumentParser(description=desc)
 parser.add_argument("-b",
                     help="Name of the branch to move to, example -b v2_devel")
-parser.add_argument("-d",
-                    help="Clear the arm home folder, remove all directories and files",
-                    action='store_true')
+# parser.add_argument("-d",
```
Comment (shitwolfymakes) [inline devtools/armdevtools.py:23]: Remove this commented code

File: devtools/armdevtools.py [hunk 2]
```diff
@@ -3,76 +3,40 @@
...
+parser.add_argument("-dr",
+                    help="Docker rebuild post ARM code update. Requires docker run script path to run.")
 parser.add_argument("-db_rem",
                     help="Database tool - remove current arm.db file",
                     action='store_true')
-parser.add_argument("-db_data",
-                    help="Database tool - populate the database with Lorem Ipsum data. " +
-                    "Requires the active database to be the most current",
-                    action='store_true')
+# parser.add_argument("-db_data",
```
Comment (shitwolfymakes) [inline devtools/armdevtools.py:31]: Review this commented code

File: devtools/armdevtools.py [hunk 3]
```diff
@@ -81,23 +45,27 @@ def arm_clear_data():
 args = parser.parse_args()
 # -b move to branch
 if args.b:
-    git_branch_change(args.b)
+    armgit.git_branch_change(args.b, arm_install)
+
+# -d Delete/Clear arm home drive data [currently unsupported]
```
Comment (shitwolfymakes) [inline devtools/armdevtools.py:50]: Remove this commented code

File: devtools/armdevtools.py [hunk 4]
```diff
@@ -81,23 +45,27 @@ def arm_clear_data():
...
+    armdocker.docker_rebuild(args.dr, arm_install)
 
 # -db_rem Database remove
 if args.db_rem:
     database.remove()
 
-# -db_data Database data insert
-if args.db_data:
-    database.data()
+# -db_data Database data insert [currently unsupported]
```
Comment (shitwolfymakes) [inline devtools/armdevtools.py:62]: Remove this commented code

File: devtools/armui.py [hunk 5]
```diff
@@ -25,9 +27,36 @@ def stop():
 #  INPUT: none
```
Comment (shitwolfymakes) [inline devtools/armui.py:27]: Remove redundant comments

**Review by shitwolfymakes — CHANGES_REQUESTED** [review]

File: devtools/armdevtools.py (hunk 2 above)
Comment (microtechno9000) [reply devtools/armdevtools.py:31]: Removed comments

File: devtools/armdevtools.py (hunk 4 above)
Comment (microtechno9000) [reply devtools/armdevtools.py]: Removed comments

File: devtools/armdevtools.py (hunk 1 above)
Comment (microtechno9000) [reply devtools/armdevtools.py]: Removed comments

File: devtools/armui.py (hunk 5 above)
Comment (microtechno9000) [reply devtools/armui.py]: Removed comments

**Review by shitwolfymakes — APPROVED** [review]
Good to go. Keep up the great work!

**Pull request merged**